### ISSUE TITLE
Add more unit test to increase the test coverage

### ISSUE THREAD

**Issue by ylwu-amzn** [opensearch-project/ml-commons#1342]
Add more unit test to increase the test coverage

**Comment by austintlee** [comment]
Can we set a specific target for this issue?  Like 65% or 70%?  We should aim to make incremental improvements.

**Comment by ylwu-amzn** [comment]
Yes, we have some todos in the code, for example this one https://github.com/opensearch-project/ml-commons/blob/main/common/build.gradle#L43
